system/lib/libcxx/include/exception

```diff
@@ -45,7 +45,11 @@ unexpected_handler get_unexpected() noexcept;
 typedef void (*terminate_handler)();
 terminate_handler set_terminate(terminate_handler  f ) noexcept;
 terminate_handler get_terminate() noexcept;
+#ifdef __USING_WASM_EXCEPTIONS__
```

Comment:
Might want to add a comment here, or just point people to libcxxabi for an explanation. Unless that's obvious enough?

Reply:
I was not sure if it would be OK to copy-paste the same comment four times. Did that anyway.